daemon/events/events_test.go

```diff
@@ -0,0 +1,117 @@
+package events
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
+)
+
+func TestBasicSubscribe(t *testing.T) {
+	type eventsOption struct {
+		action    string
+		eventType types.EventType
+		actor     *types.EventsActor
+	}
+
+	ctx := context.Background()
+	testEvents := []eventsOption{
+		{
+			action:    "create",
+			eventType: types.EventTypeContainer,
+			actor:     &types.EventsActor{ID: "asdf"},
+		},
+		{
+			action:    "create",
+			eventType: types.EventTypeContainer,
+			actor: &types.EventsActor{
+				ID: "qwer",
+				Attributes: map[string]string{
+					"image": "busybox",
+				},
+			},
+		},
+	}
+
+	eventsService := NewEvents()
+
+	t.Log("subscribe")
+	var cancel1, cancel2 func()
```

Comment:
since the following statements use `:=`, it seems this line is useless here. how do you think?